### ISSUE TITLE
[CI] SearchWithRandomIOExceptionsIT#testRandomDirectoryIOExceptions failure

### ISSUE THREAD

**Issue by martijnvg** [elastic/elasticsearch#25764]
Test failed because suite timeout was exceeded:

```
ERROR   0.00s J0 | SearchWithRandomIOExceptionsIT (suite) <<< FAILURES!
   > Throwable #1: java.lang.Exception: Suite timeout exceeded (>= 1200000 msec).
   > 	at __randomizedtesting.SeedInfo.seed([22D77D0ABF822429]:0)
```

Build url: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+g1gc/3577/consoleFull

There is also an uncaught seqno related assertion failure: 

```
Captured an uncaught exception in thread: Thread[id=2138, name=elasticsearch[node_s0][refresh][T#1], state=RUNNABLE, group=TGRP-SearchWithRandomIOExceptionsIT]
   > Caused by: java.lang.AssertionError
   > 	at __randomizedtesting.SeedInfo.seed([22D77D0ABF822429]:0)
   > 	at org.elasticsearch.index.seqno.GlobalCheckpointTracker.updateLocalCheckpoint(GlobalCheckpointTracker.java:476)
   > 	at org.elasticsearch.index.seqno.SequenceNumbersService.updateLocalCheckpointForShard(SequenceNumbersService.java:141)
   > 	at org.elasticsearch.index.shard.IndexShard.updateLocalCheckpointForShard(IndexShard.java:1664)
   > 	at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryShardReference.updateLocalCheckpointForShard(TransportReplicationAction.java:997)
   > 	at org.elasticsearch.action.support.replication.ReplicationOperation.execute(ReplicationOperation.java:108)
   > 	at org.elasticsearch.action.support.replication.TransportReplicationAction$AsyncPrimaryAction.onResponse(TransportReplicationAction.java:338)
   > 	at org.elasticsearch.action.support.replication.TransportReplicationAction$AsyncPrimaryAction.onResponse(TransportReplicationAction.java:282)
   > 	at org.elasticsearch.action.support.replication.TransportReplicationAction$1.onResponse(TransportReplicationAction.java:927)
   > 	at org.elasticsearch.action.support.replication.TransportReplicationAction$1.onResponse(TransportReplicationAction.java:924)
   > 	at org.elasticsearch.index.shard.IndexShardOperationPermits.acquire(IndexShardOperationPermits.java:235)
   > 	at org.elasticsearch.index.shard.IndexShard.acquirePrimaryOperationPermit(IndexShard.java:2091)
   > 	at org.elasticsearch.action.support.replication.TransportReplicationAction.acquirePrimaryShardReference(TransportReplicationAction.java:936)
   > 	at org.elasticsearch.action.support.replication.TransportReplicationAction.access$400(TransportReplicationAction.java:94)
   > 	at org.elasticsearch.action.support.replication.TransportReplicationAction$AsyncPrimaryAction.doRun(TransportReplicationAction.java:299)
   > 	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)
   > 	at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryOperationTransportHandler.messageReceived(TransportReplicationAction.java:278)
   > 	at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryOperationTransportHandler.messageReceived(TransportReplicationAction.java:265)
   > 	at org.elasticsearch.transport.AssertingTransportInterceptor$1.messageReceived(AssertingTransportInterceptor.java:76)
   > 	at org.elasticsearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:66)
   > 	at org.elasticsearch.transport.TransportService$7.doRun(TransportService.java:650)
   > 	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:638)
   > 	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)
   > 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
   > 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
   > 	at java.lang.Thread.run(Thread.java:748)
```

**Comment by bleskes** [comment]
@jasontedor I understand you're already looking at this. Let me know if that's incorrect...

**Comment by jasontedor** [comment]
Closed by fd5710195212534c260e2d496adee26e12f190f7 and #25822
